### PR TITLE
CRS-603 (Temporary) Add bulk comparison functionality to aid testing of the algorithm. This facilitates testing of the algorithm and compares the results to NOMIS

### PR DESCRIPTION
**Pull request by yasinmustafa**
This is just a temporary piece of functionality to aid testing of the calculation algorithm compared with NOMIS. It allows testing of multiple prisoners at a time and generates a CSV file showing the differences between what we calculated and the NOMIS calculation (which could be an overriden value)

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -16,7 +16,8 @@
     "allowSyntheticDefaultImports": true,
     "noImplicitAny": true,
     "experimentalDecorators": true,
-    "typeRoots": ["./server/@types", "./node_modules/@types"]
+    "typeRoots": ["./server/@types", "./node_modules/@types"],
+    "lib": ["ES2020.Promise"]
```
Comment (yasinmustafa) [inline tsconfig.json:20]: I added this in as it allows use of the `Promise.allSettled()` - which is useful when stacking api calls and handling individual failures. Didnt use it in the end as I went down the synchronous route

File: server/routes/index.ts [hunk 2]
```diff
@@ -48,7 +48,8 @@ export default function Index({
   }
 
   const otherRoutes = () => {
-    get('/test/calculation', otherAccessRoutes.testCalculation) // TODO remove this route as it was only for POC testing
```
Comment (ldlharper) [inline server/routes/index.ts:51]: Did we want to add something here to stop this page being accessible on prod?

**Review by ldlharper — APPROVED** [review]
LGTM. Just 1 minor comment.

File: server/routes/index.ts (hunk 2 above)
Comment (yasinmustafa) [reply server/routes/index.ts:51]: we might do - not just yet. Will discuss further - we're thinking it might be useful in prod.

**Pull request merged**